packages/react-search-ui/src/containers/__tests__/__snapshots__/ResultsPerPage.test.tsx.snap

```diff
@@ -50,12 +51,7 @@ exports[`renders the component with custom page options 1`] = `
         "valueContainer": [Function],
       }
     }
-    value={
-      Object {
-        "label": 10,
-        "value": 10,
-      }
-    }
+    value={null}
   />
```

Comment:
Changes in this file worry me. Could you check if all the changes in the component are intended? It seems that we replaced `value` with `resultsPerPage`.

Reply:
Also 
```
Object {
  "label": 20,
  "value": 20,
}
```
Turned into just 
```
20
```

Reply:
thanks for this spot! The issue was down to a refactor with the value props and didn't finish it.